### PR TITLE
Adds in a Jug of BananaHonk

### PR DESCRIPTION
**Pull request by shazbot194**
After talking it over with Driker, we will no longer be adding in a new drink but rather just adding in the jug and having it filled with BananaHonk. I will update the code to reflect this as soon as I can.

et8nu6v

:cl: Shazbot-coder, Driker-spriter/idea_owner
Adds: Gives the clown a jug of BananaHonk.
/:cl:

### CODE REVIEW

File: code/modules/reagents/chemistry/reagents/drinks.dm [hunk 1]
```diff
@@ -167,6 +167,20 @@
 		M.adjustFireLoss(-1)
 	..()
 
+/datum/reagent/consumable/drink/jolly
+	name = "Jolly Juice"
+	id = "jolly"
+	description = "Juice that make you go HONK!!"
```
Comment (SamHPurp) [inline code/modules/reagents/chemistry/reagents/drinks.dm]: `that'll make` or `that makes` - grammatical issue

File: code/modules/reagents/chemistry/reagents/drinks.dm [hunk 2]
```diff
@@ -167,6 +167,20 @@
...
+	drink_icon = "jolly_juice"
+	drink_name = "Jolly Juice"
+	drink_desc = "HOINK!!!"
```
Comment (SamHPurp) [inline code/modules/reagents/chemistry/reagents/drinks.dm]: `hoink`?

Comment (shazbot194) [inline code/modules/reagents/chemistry/reagents/drinks.dm]: You know what I'm going to say, sleep needed.

**Comment by SamHPurp** [comment]
> Grammer fix

> Grammer 

🤣

**Comment by uraniummeltdown** [comment]
Don't change the bananium grinding reagent to jolly juice, leave it as banana juice

**Comment by Fox-McCloud** [comment]
I'm all for silly stuff, but as @uraniummeltdown points out; this is wholly redundant.

It's not really a balance implication, nor is it particularly problematic (oh no, clown is going to robust us all with banana juice). It's just making something harder to get for...vague reasons? I really don't understand the motivations behind the PR, to be honest.

The bottle sprite would be neat for a pre-spawned amount of Banana Honk, but that's about it, IMO.

**Comment by taukausanake** [comment]
@Fox-McCloud I think the motivation is to make the clown more similar to the mime as said in OP, "clown's version of nothing". I might agree with you and Uranium though that it isn't really needed but wouldn't be bad for the clown to have his own drink. How much is in the Bottle of Nothing anyways? (dumb question but you get what I mean) If it's an unlimited drink then maybe the clown can have a similar drink container?

**Comment by The-Driker** [comment]
@uraniummeltdown 
The entire point is to give the clown a starting drink item that matches up with the mime, but flavored specifically to fit the clown. 
As it says in the pr itself, it removes the healing effect the clown gets from banana juice and gives it to this drink instead. 
@Fox-McCloud 
It's not so much harder to get, as it is equal in accessibility as the Mime's “Nothing”.  The clown would start with a jug, and just like the mime, could get it refilled at a hacked soda machine. 
@taukausanake 
Yep! It's not unlimited, that would be a bit unbalanced. Just like the Mime's "Nothing", it's 100 units and it heals 1 brute/1burn.

**Comment by Necaladun** [comment]
Love the sprite, would love to see the clown get something like this. 

However, I'd prefer they not start with it, and have an empty/nearly empty jug in their office. Ground bananium seems a bit weird to me tbh, I think just banana juice + something else funny (space lube?) would be enough.

The reason I don't want it round start is because self-healing is pretty powerful, and especially round start before the engine is up can be a decent replacement for insulated gloves/heals/etc. The clown should not be able to shrug off robusting and the consequences of dangerous pranks/shittery without co-operation from others. Having to give up bananas to be juiced, work with the barman/hydro, etc, seems a good way of going about that.

**Comment by taukausanake** [comment]
@Necaladun Doesn't sound very fair then that the mime gets round start healing when it will get robusted a lot less than the clown. I think this is a good idea but just needs some tweaking

**Comment by The-Driker** [comment]
@Necaladun
The Mime's bottle of Nothing does exactly this already. It's a 100 unit bottle, that heals 1 brute/burn.

**Comment by Jountax** [comment]
@taukausanake The bottles of nothing (there's a second one in the mime's locker) have a limited amount of the reagent named "nothing"

To my knowledge the only source of more nothing is mimanas from botany.

**Comment by The-Driker** [comment]
@Jountax 

You can multi-tool the soda fountain to get more Nothing.

**Comment by uraniummeltdown** [comment]
@The-Driker Banana juice is already the clown's reagent, jolly juice is totally unnecessary
Why not just add a glass/bottle of banana juice to the hacked soda machine? And I really don't like adjusting every instance of banana juice to be "jolly juice". You removed banana juice's clown healing and gave it to jolly juice. What is the point of adding jolly juice when banana juice is already the clown healing drink?

Seriously just add the jug of banana juice, not this copy-paste reagent

**Comment by taukausanake** [comment]
The only reason I could see to change banana juice or have a different reagent would be to keep it on par with the mime's Nothing reagent. At that point however you might as well modify Nothing reagent instead. Is that something that might be needed if to keep them similar?

**Comment by uraniummeltdown** [comment]
They're already similar, Nothing and Banana Juice heal at the same rate. Nothing heals mimes only, Banana Juice heals Clowns and monkeys only.

**Comment by shazbot194** [comment]
> They're already similar, Nothing and Banana Juice heal at the same rate. Nothing heals mimes only, Banana Juice heals clowns and monkeys only.

Yes, they are similar, but "nothing" is harder to get, and comes in its own container. This would help level them out, making them equally hard to get, and give both of them a bottle of their base drinks round start. I am also making a new banana similar to the mimana which gives Jolly juice and lube as opposed to the mimana which gives nothing and Capulettium Plus.
In addition, you can get "nothing" from hacked drink machine, but you can get banana juice from a default soda machine.

**Comment by scrubmcnoob** [comment]
Maybe just give the clown a bottle called the Jolly Bottle with this sprite full of banana juice?

**Comment by uraniummeltdown** [comment]
@shazbot194 Not everything needs "balancing" you know, when you play mime or clown the first thing that comes to mind is not healing with a special drink, they're not combat roles
Bottle of Nothing is hardly used for healing, most mimes use it as a decent brute weapon. It fits the mime to have a bottle because he's French and therefore drinks a lot

If you must, give the Clown a bottle of banana juice with your sprite. We don't need a new banana specifically for a snowflake juice which is an exact copy of an already existing one

**Comment by Vivalas** [comment]
We don't really need a special reagent for the clown when we already have one, banana juice. Is it unique and hard to get? Not exactly, but it doesn't have to line up exactly with the mime's reagent.

I prefer the more supported option of giving the clown a bottle of banana juice and adding banana juice to hacked vendors.

**Comment by taukausanake** [comment]
There really should be a better way for mime to get Nothing. You talk about not needing to balance clown and mime but you're okay with giving the clown a bottle with healing juice that's easy enough to get compared to what the mime has. Make Nothing a little easier to get, maybe the mime can generate it himself from an invisible drink dispenser or default from the bartender. It's only fair

**Comment by Fox-McCloud** [comment]
changes necessary to get this merged: No new reagent.

We're ok with there being a pre-spawned bottle (that uses this sprite) that's filled with Banana Juice, but beyond that, we don't feel it's necessary for a completely new reagent.

**Comment by shazbot194** [comment]
After talking it over with Driker, we have come to the conclusion that we will have the jug filled with BananaHonk and not add in Jolly Juice. I will update the PR tomorrow when I have access to the computer I make my pulls on.

**Comment by Fox-McCloud** [comment]
banana juice would be more accurate; it's the "unrefined" version of bananahonk...just like "nothing" is unrefined "silencer" in a way.

**Comment by shazbot194** [comment]
So my master branch has exploded and due to the way I use github, this has made it impossible to update this pull, so I will be remaking it tonight.

**Pull request closed without merging**